From linux@roeck-us.net  Tue May  5 23:59:20 2015
From: Guenter Roeck <linux@roeck-us.net>
Date: Mon, 04 May 2015 21:42:41 -0700
Subject: s390: Fix build error
To: Greg Kroah-Hartman <gregkh@linuxfoundation.org>, linux-kernel@vger.kernel.org
Message-ID: <55484A41.1090906@roeck-us.net>


From: Guenter Roeck <linux@roeck-us.net>

s390 images fail to build in 3.10 with

arch/s390/kernel/suspend.c: In function 'pfn_is_nosave':
arch/s390/kernel/suspend.c:147:10: error: 'ipl_info' undeclared
arch/s390/kernel/suspend.c:147:27: error: 'IPL_TYPE_NSS' undeclared

due to a missing include file.

Signed-off-by: Guenter Roeck <linux@roeck-us.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 arch/s390/kernel/suspend.c |    1 +
 1 file changed, 1 insertion(+)

--- a/arch/s390/kernel/suspend.c
+++ b/arch/s390/kernel/suspend.c
@@ -11,6 +11,7 @@
 #include <linux/mm.h>
 #include <asm/sections.h>
 #include <asm/ctl_reg.h>
+#include <asm/ipl.h>
 
 /*
  * The restore of the saved pages in an hibernation image will set
